pkg/controller/supportbundlecollection/controller.go

```diff
@@ -391,12 +391,16 @@ func (c *Controller) createInternalSupportBundleCollection(bundle *v1alpha1.Supp
 	}
 	nodeSpan := nodeNames.Union(externalNodeNames)
 	// Get authentication from the Secret provided in authentication field in the CRD
-	authentication, err := c.parseBundleAuth(bundle.Spec.Authentication)
+	authentication, err := ftp.ParseFileServerAuth(ftp.FileServerAuthType(bundle.Spec.Authentication.AuthType), bundle.Spec.Authentication.AuthSecret, c.kubeClient)
```

Comment:
The line between 62-65 should be removed since it's already defined in auth.go

Reply:
done